src/hooks/useClubIntegrations.ts

```
@@ -0,0 +1,153 @@
+// =============================================================================
+// Hook: useClubIntegrations
+// Issue: #3542 - Implement 'Automated Multi-Channel Cross-Posting'
+// Description: Manages the CRUD operations for club webhooks.Allows admins 
+// to add Discord / Slack URLs, toggle their active status, and trigger test
+// pings to verify the connection before an event goes live.
+    // =============================================================================
+
+    import { useState, useEffect, useCallback } from 'react';
```

ESLint

[error] Replace `····import·{·useState,·useEffect,·useCallback·}·from·'react'` with `import·{·useState,·useEffect,·useCallback·}·from·"react"`

(prettier/prettier)